Documentation/Books/Manual/Programs/Arangodump/Maskings.md

```diff
@@ -131,12 +279,143 @@ Note that the hash is based on a random secrect that is different for
 each run. This avoids dictionary attacks.
 
 If you need reproducable results, i.e. hash that do not change between
-different runs of *arangodump*, you need to specify a seed, which must
-not be `0`.
+different runs of *arangodump*, you need to specify a secret as seed,
+a number which must not be `0`.
 
     {
+      "type": "xify_front",
       "path": ".name",
       "unmaskedLength": 2,
       "hash": true,
       "seed": 246781478647
     }
+
+### zip (Enterprise)
+
+This masking replaces a zip code with a random one. If the attribute value
+is not a string it is replaced by `12345`.
+
+    {
+      "path": ".code",
+      "type": "zip",
+    }
+
+This will replace an existing zip with a random one. It uses the following
+rule: If a character of the original zip code is a digit it will be replaced
+by a random digit. If a character of the original zip code is a letter it
+will be replaced by a random letter keeping the case.
+
+    {
+      "path": ".code",
+      "type": "zip",
+      "default": "abcdef"
+    }
+
+If the attribute value is not a string use the value of default `abcdef`.
+
+#### Example
+
+If the original zip code is
+
+    50674
+
+it will be replaced by (for example)
+
+    98146
+
+If the original zip code is
+
+    SA34-EA
+
+it will be replaced by (for example)
+
+    OW91-JI
+
+Please note that this will generate random zip. Therefore there is a chance
+that it will destroy uniqueness.
+
+### date-time (Enterprise)
+
+This masking replaces the value of the attribute with a random date.
+
+    {
+      "type": "date-time",
+      "begin" : "2019-01-01",
+      "end": "2019-12-31",
+      "format": "%yyyy-%mm-%dd",
+    }
+
+`begin` and `end` are in ISO8601.
+
+The format is described in
+[DATE_FORMAT](https://docs.arangodb.com/3.4/AQL/Functions/Date.html#dateformat).
+
+### integral number (Enterprise)
+
+This masking replaces the value of the attribute with a random integral number.
```

Comment:
Always, or only if the attribute type is number?

Reply:
done